english/33hours/pillow.md

```diff
@@ -0,0 +1,3 @@
+As you lift the pillow to see what is underneath, the strong smell of dried blood caking on top nauseates you. You feel dizzy again. But you sense something metallic under the pillow. You want to check it out, but your eyes are closing. It is getting darker. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.